test/jdk/javax/swing/text/html/HTMLEditorKit/bug4245401.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 1999, 2025, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/* @test
+ * @bug 4245401
+ * @summary Tests that JTextPane with HTMLEditorKit handles the HEAD tag properly
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
+ * @run main/manual bug4245401
+ */
+
+import java.io.StringReader;
+import javax.swing.JFrame;
+import javax.swing.JTextPane;
+import javax.swing.text.html.HTMLDocument;
+import javax.swing.text.html.HTMLEditorKit;
+
+public class bug4245401 {
+
+    static final String INSTRUCTIONS = """
+        Place the cursor before the "H" in the word "HTML"
+        Then press <Up-arrow> and <Enter>.
+        If an extra HEAD tag appear, the test FAILS, otherwise it PASSES.
+    """;
+
+    static JFrame createUI() {
+
+        JFrame frame = new JFrame("bug4245401");
+        JTextPane ep = new JTextPane();
+        ep.setEditable(true);
+        ep.setContentType("text/html");
+        HTMLEditorKit kit = (HTMLEditorKit) ep.getEditorKit();
+        ep.setEditorKit(kit);
+        HTMLDocument doc = (HTMLDocument) kit.createDefaultDocument();
+        ep.setDocument(doc);
+
+        try {
+            String text = "HTML Test... Test is a test...";
+            kit.read(new StringReader(text), doc, 0);
+        } catch (Exception e) {
+        }
```

Comment:
If an exception occurs, the test should fail; the exception shouldn't be *silently* ignored.

Reply:
Not something I added but I can re-throw as RTE. I verified that will cause the test to fail.

Reply:
Thank you.